06 week/homework/game.js

```diff
@@ -0,0 +1,115 @@
+// 1. We listen for the arrow events and move the player
+let player = document.querySelector(".player");
+
+document.addEventListener("keydown", function (event) {
+    let oldTop = parseInt(player.style.top)
+    let oldLeft = parseInt(player.style.left)
```

Comment:
era interesat daca ai fi mutat logica in functie mai mici + cea de verificare si sa tot aici facut prin utilizarea de functii